task_10/index.html

```diff
@@ -0,0 +1,92 @@
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto">
+    <link rel="stylesheet" type="text/css" href="css/reset.css">
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+    <script type="text/javascript" src="js/config.js"></script>
+    <script type="text/javascript" src="js/helper.js"></script>
+    <script type="text/javascript" src="js/models/model.js"></script>
+    <script type="text/javascript" src="js/views/view.js"></script>
+    <script type="text/javascript" src="js/services/weather_service.js"></script>
+    <script type="text/javascript" src="js/services/storage_service.js"></script>
+    <script type="text/javascript" src="js/services/list_service.js"></script>
+    <script type="text/javascript" src="js/services/favorites_service.js"></script>
+    <script type="text/javascript" src="js/services/history_service.js"></script>
+    <script type="text/javascript" src="js/controllers/controller.js"></script>
+    <script type="text/javascript" src="js/app.js"></script>
+    <title>Weather App</title>
+  </head>
+  <body>
+    <header class="header">
+      <div class="flex-container">
+          <span>
+            <button class="btn" id="add-favorite-btn" title="Adds city to favorites">
+              <i class="fa fa-star" aria-hidden="true"></i>
+            </button>
+          </span>
+          <span class="loc-field">
+            <input class="btn wide" id="loc-field" type="text" placeholder="Enter location..." title="Enter location here">
+          </span>
+          <span class="base-units">
+            <select class="btn" id="base-units" title="Selects metric or imperial units">
+              <option value="metric">&deg;C, m/s</option>
+              <option value="imperial">&deg;F, mph</option>
+            </select>
+          </span>
+      </div>
+    </header>
+    <main>
+      <section>
+        <div class="flex-container current-day" id="current-day"></div>
+      </section>
+      <section>
+        <div class="flex-container another-days" id="another-days"></div>
+      </section>
+    </main>
+    <footer class="flex-container footer">
+      <div class="favorites">
+        <span>
+          <i class="fa fa-star" aria-hidden="true"></i>
+        </span>
+        <span>
+          <input class="btn" id="favorites-field" type="text" list="favorites-lst" placeholder="Favorites..." title="List of favorites">
+          <datalist id="favorites-lst"></datalist>
+        </span>
+        <span>
+          <button class="btn" id="favorites-go-btn" title="Loads selected city data">
+            <i class="fa fa-search" aria-hidden="true"></i>
+          </button>
```

Comment:
Button should provide some text readable by screen readers. Read more about _screenreader-only_ hack and provide captions to buttons.

Also, this help people who read the code understand what is going on; because `fa-search` may mean more than one thing.

Reply:
Aria-labels were added to all of the buttons without inner text.